activities/VideoViewer.activity/lib/tutorial.js

```diff
@@ -0,0 +1,86 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
+            {
+                element: "",
+                orphan: true,
+                placement: "bottom",
+                title: l10n.get("TutoExplainTitle"),
+                content: l10n.get("TutoExplainContent")
+            },
+            {
+                element: "#filter-button",
+                placement: "bottom",
+                title: l10n.get("TutoFilterTitle"),
+                content: l10n.get("TutoFilterContent")
+            },
+            {
+                element: "#favorite-button",
+                title: l10n.get("TutoFavoriteTitle"),
+                content: l10n.get("TutoFavoriteContent")
+            },
+            {
+                element: "#library-button",
+                title: l10n.get("TutoLibraryTitle"),
+                content: l10n.get("TutoLibraryContent")
+            },
+            {
+                element: "#exportvideo-button",
+                title: l10n.get("TutoExportvideoTitle"),
+                content: l10n.get("TutoExportvideoContent")
+            },
+            {
+                element: "#search",
+                title: l10n.get("TutoSearchTitle"),
+                content: l10n.get("TutoSearchContent")
+            },
+            {
+                element: "#app_item",
+                title: l10n.get("TutoVideoideoTitle"),
```

Comment:
Look like this string is not defined in the locale.ini. Is a typo?

Reply:
Oh, yeah, you are right. It's a typo. I'll correct that in a second.